crates/subspace-farmer/src/bench_rpc_client.rs

```diff
@@ -0,0 +1,201 @@
+use std::sync::Arc;
```

Comment:
Are you sure this should even be a part of the library? I think you could have added this to the CLI crate instead and not expose it from the library.

Reply:
I thought it would be usefull to reuse it from the gui, no?

Reply:
We can make it separate crate though

Reply:
Let's just move it from library into binary for now. We have created `RpcClient` specifically such that we can implement it externally if need be.